src/JBDistributor.sol

```diff
@@ -1,208 +1,286 @@
+    
+    //*********************************************************************//
+    // --------------------- public stored properties -------------------- //
+    //*********************************************************************//
+
+    /// @notice The amount of a token that is currently vesting.
+    /// @custom:param token The address of the token that is vesting.
+    mapping(IERC20 token => uint256) public vestingAmountOf;
+
+    /// @notice The snapshot data of the token information for each round.
+    /// @custom:param token The address of the token being claimed and vested.
+    /// @custom:param round The round to which the data applies.
+    mapping(IERC20 token => mapping(uint256 => TokenSnapshotData)) public snapshotAtRoundOf;
+
+    /// @custom:param tokenId The ID of the token to which the vesting amount belongs. 
+    /// @custom:param round The round during which the vesting began. 
+    /// @custom:param token The address of the token being vested. 
+    mapping(uint256 => mapping(uint256 => mapping(IERC20 => uint256))) public vestingTokenAmountAtRoundOf;
+
+    //*********************************************************************//
+    // -------------------------- public views --------------------------- //
+    //*********************************************************************//
+      
+    /// @notice The number of the current round.
+    function currentRound() public view returns (uint256) {
+        return (block.number - startingBlock) / roundDuration;
+    }
+    
+    /// @notice The block at which a round started.
+    /// @param _round The round to get the start block of.
+    function roundStartBlock(uint256 _round) public view returns (uint256) {
+        return startingBlock + roundDuration * _round;
+    }
 
-    // The amount of a token that is currently vesting
-    mapping(IERC20 => uint256) public tokenVestingAmount;
-
-    // The snapshot data of the token information for each round
-    // IERC20 -> cycle -> token information
-    mapping(IERC20 => mapping(uint256 => TokenState)) public tokenAtRound;
-
-    // Maps tokenId -> cycle -> IERC20 token -> release amount
-    mapping(uint256 => mapping(uint256 => mapping(IERC20 => uint256))) public tokenVesting;
+    //*********************************************************************//
+    // -------------------------- constructor ---------------------------- //
+    //*********************************************************************//
 
-    /**
-     *
-     * @param _roundDuration The duration of a period/cycle in blocks (IMPORTANT: make sure this is correct for each blockchain/rollup this gets deployed to)
-     * @param _vestingRounds The number of cycles it takes for rewards to vest
-     */
+    /// @param _roundDuration The minimum amount of time stakers have to claim rewards, specified in blocks. (IMPORTANT: make sure this is correct for each blockchain/rollup this gets deployed to)
```

Comment:
"minimum amount of time (...) specified in blocks" -> "the minimum number of blocks during which (...)" ?

Reply:
nice, yes. send the fix :salute: